### ISSUE TITLE
Omit certain fields from autogeneration in `getMockHandlerInput`

### ISSUE THREAD

**Issue by itslorant** [swarmion/swarmion#647]
### Clear and concise description of the problem

The `getMockHandlerInput` fill up the contract's fields as intended, but some cases it would be great if we can define attributes that we do not want to fill with fake data. (The data can be in the body or in the query strings parameters...)

Why:
- For my case there is some `if` in the handler which manipulates the data according from the input. It can not 
work around every option with a different schema.

### Suggested solution

The `getMockHandlerInput` has an optional array, where can be defined which fields we do not want to populate.

For example:
```typescript
    getMockHandlerInput(myContract, {
      body:{
        name: 'John'
      },
      excludeFields:['address', 'phoneNumber']
    }
```

### Alternative

I tried the `undefined` that does not worked. Other idea was the `null`, but my case the field's value can not be `null`.

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/swarmion/swarmion/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://www.swarmion.dev/docs/).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Sc0ra** [comment]
Hello

What was the problem you encountered using undefined ? It is indeed how I would have proceeded.
Could you please give me the contract and its schemas so I can reproduce it more precisely ?

Thanks !

**Comment by itslorant** [comment]
Sorry i wasn't clear enough.

> What was the problem you encountered using undefined ? 

I tried this:
```typescript 
const eventData = getMockHandlerInput(personContract, {
      body: {
        name: 'John',
        email: 'john@example.com',
        address: undefined,
      },
    });
```
but the generated data has an address field with fake data.

> Could you please give me the contract and its schemas so I can reproduce it more precisely ?

Here is a little example:

Contract:
```typescript

export const personSchema = {
  type: 'object',
  properties: {
    name: { type: 'string' },
    title: { type: 'string' },
    address: { type: 'string' },
    email: { type: 'string' },
    alias: { type: 'string' },
  },
  required: ['name', 'email'],
  additionalProperties: false,
} as const;

export const personContract = new ApiGatewayContract({
  id: 'person',
  path: '/person',
  method: 'POST',
  integrationType: 'restApi',
  outputSchemas: {
    [HttpStatusCodes.OK]: {},
  },
  bodySchema: personSchema,
});
```

Handler:
```typescript
export const main = getHandler(personContract, { ajv })(async event => {
  if (event.body.address == null) {
    //do something
  }

  if (event.body.alias == null) {
    //do something
  }
  await Promise.resolve('ok');

  return {
    statusCode: 200,
    body: {},
  };
});
```

In the test case:
```typescript
 const eventData = getMockHandlerInput(personContract, {
      body: {
        name: 'John',
        email: 'john@example.com',
      },
    });
```

In this case the `getMockHandlerInput` will create this:
```typescript
EventData [
  {
    pathParameters: undefined,
    queryStringParameters: undefined,
    headers: undefined,
    body: '{"name":"John","email":"john@example.com","address":"aliquip sed officia","title":"dolor consequat aliqua labore","alias":"pariatur ex laborum id laboris"}',
    requestContext: {
      accountId: '',
      apiId: '',
      authorizer: undefined,
      protocol: '',
      httpMethod: '',
      identity: [Object],
      path: '',
      stage: '',
      requestId: '',
      requestTimeEpoch: 0,
      resourceId: '',
      resourcePath: ''
    }
  },
  {
    callbackWaitsForEmptyEventLoop: false,
    functionName: '',
    functionVersion: '',
    invokedFunctionArn: '',
    memoryLimitInMB: '',
    awsRequestId: '',
    logGroupName: '',
    logStreamName: ''
  },
  [Function (anonymous)]
]
```

And my thought was if we can pass down an extra array or object like this:
```typescript
const eventData = getMockHandlerInput(personContract, {
      body: {
        name: 'John',
        email: 'john@example.com',
      },
      excludeFields: ['address']
    });
```

then the generated data should like something like this:
(without the address)
```typescript
{
    pathParameters: undefined,
    queryStringParameters: undefined,
    headers: undefined,
    body: '{
      "name":"John",
      "email":"john@example.com",
      "title":"dolor consequat aliqua labore",
      "alias":"pariatur ex laborum id laboris"
      }',
}
```

**Comment by Sc0ra** [comment]
Hello @itslorant 
I just opened a PR with a fix for this problem #648 

For me, the right way to do so was to use undefined correctly.
Ask me anything if you have question about this problem.

We'll ping you back when we release this fix (we hope to do it this afternoon).

**Comment by itslorant** [comment]
Hello @Sc0ra 
This is looks great!
Thank you!
